### PR TITLE
Add Scottish Resistance sticky outline fix

### PR DESCRIPTION
**Pull request by treacherousfiend**
Its a relatively common question, and considering it can be fixed with modules I feel like it should be mentioned either here or on the modules page.

### CODE REVIEW

**Comment by Tiagoquix** [comment]
Does this fix work for DX8 users?

**Comment by treacherousfiend** [comment]
I had no clue there were even issues with the SR on DX8, i'll have to do some testing

File: docs/next_steps/quick_fixes.md [hunk 1]
```diff
@@ -129,3 +129,7 @@ cl_interp x
 ```
 
 One other thing to note is that client interp is determined by whichever is the greater of cl_interp_ratio divided by cl_updaterate, and the value of cl_interp. cl_interp_ratio and cl_interp have no interaction beyond that.
+
+## Scottish Resistance stickybombs have no outline
+
+Due to a bug in the game, the outlines for Scottish Resistance stickybombs only appears when objective outlines are enabled. You can fix this by adding `outlines=medium` to your `modules.cfg`.
```
Comment (mastercoms) [inline docs/next_steps/quick_fixes.md]: It's not a bug, `glow_outline_effect_enable` controls all outline effects. I would just say that SR stickies require outlines, which can be enabled with at least outlines=medium module level.

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: docs/next_steps/quick_fixes.md (hunk 1 above)
Comment (treacherousfiend) [reply docs/next_steps/quick_fixes.md]: Should be better now? i'll let you resolve if you think its good.
I might've overdid it though, i'm not great at this sort of thing.

Comment (Tiagoquix) [reply docs/next_steps/quick_fixes.md]: So this won't work on DirectX 8.

@treacherousfiend Please add a note about this.

File: docs/next_steps/quick_fixes.md [hunk 2]
```diff
@@ -129,3 +129,7 @@ cl_interp x
 ```
 
 One other thing to note is that client interp is determined by whichever is the greater of cl_interp_ratio divided by cl_updaterate, and the value of cl_interp. cl_interp_ratio and cl_interp have no interaction beyond that.
+
+## Scottish Resistance stickybombs have no outline
+
+On some Mastercomfig presets, outlines are disabled. The outline settings include player and objective outlines as well as outlines for Scottish Resistance stickybombs. You can enable outlines by adding `outlines=medium` to your `modules.cfg`.
```
Comment (mastercoms) [inline docs/next_steps/quick_fixes.md]:
```suggestion
On some mastercomfig presets, outlines are disabled. The outline settings include player and objective outlines as well as outlines for Scottish Resistance stickybombs. You can enable outlines by adding `outlines=medium` (or higher) to your `modules.cfg`.
```

**Review by mastercoms — CHANGES_REQUESTED** [review]

**Review by mastercoms — DISMISSED** [review]

File: docs/next_steps/quick_fixes.md (hunk 1 above)
Comment (mastercoms) [reply docs/next_steps/quick_fixes.md]: @treacherousfiend Please add a note about DX8 (in a new PR)

**Pull request merged**